thirdparty/build-thirdparty.sh

```diff
@@ -514,14 +514,26 @@ build_rocksdb() {
     export CFLAGS=$OLD_FLAGS
 }
 
+# sasl
+build_sasl() {
+    check_if_source_exist $SASL_SOURCE
+    cd $TP_SOURCE_DIR/$SASL_SOURCE
+    OLD_FLAGS=$CFLAGS
+    unset CFLAGS
+    ./autogen.sh --prefix=$TP_INSTALL_DIR --enable-gssapi=no --enable-static=yes --enable-shared=no
+    make -j$PARALLEL
+    make install
+    export CFLAGS=$OLD_FLAGS
+}
+
 # librdkafka
 build_librdkafka() {
     check_if_source_exist $LIBRDKAFKA_SOURCE
 
     cd $TP_SOURCE_DIR/$LIBRDKAFKA_SOURCE
 
     $CMAKE_CMD -DCMAKE_LIBRARY_PATH=$TP_INSTALL_DIR/lib -DCMAKE_INCLUDE_PATH=$TP_INSTALL_DIR/include \
-        -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR -DRDKAFKA_BUILD_STATIC=ON -DWITH_SASL=OFF -DWITH_SASL_SCRAM=ON \
+        -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR -DRDKAFKA_BUILD_STATIC=ON -DWITH_SASL=ON -DWITH_SASL_SCRAM=ON \
```

Comment:
What's the usage of SASL for librdkafka?

Reply:
> What's the usage of SASL for librdkafka?

To support more authentication methods